Editor/Resources/EditorWindow/Pages/AnywherePage.uss

```diff
@@ -0,0 +1,9 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+@import "../common.uss";
+
+.ip-spacer {
+    margin: 0 4px 0 4px;
+    padding-top: 8px;
+}
```

Comment:
Do you mind adding a screenshot for the Anywhere tab page?

Is this the only spot where spacing between elements is required? Wondering if this can be made general purpose to adding padding between any 2 elements, and moved to `common.uss`

Reply:
screenshot added. This is the only place where this specific styling is needed to match the mocks. Otherwise we have generic class `separator` to handle spacing between elements